src/main/java/cms/gov/madie/measure/resources/LoggingController.java

```diff
@@ -0,0 +1,35 @@
+package cms.gov.madie.measure.resources;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import io.micrometer.core.instrument.util.StringUtils;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+@RestController
+@RequiredArgsConstructor
+@Slf4j(topic = "logging")
+public class LoggingController {
+
+  @PostMapping("/log/login")
```

Comment:
could this be a generic logging controller? like just a single endpoint that takes the message and action name(login, logout, etc)? or is that a bad idea?

Reply:
@adongare Thanks Ashok! It's both login and logout, but login eventually will have successful login and unsuccessful login, that's why I kept it separate. Let me know if you still want them separate. Thank you!